rimage/jpeg.go

```diff
@@ -1,26 +0,0 @@
-//go:build no_cgo
-
-package rimage
```

Comment:
Checked that these aren't used in `rtsp` `app` or `goutils`. I am okay with keeping the non libjpeg one though @bhaney, if the convert to RGBA conversion is necessary, but it seems to be special cased to the jpeg file in `image_file` only.